api/controller.py

```diff
@@ -803,9 +803,7 @@ def crawlable_library_feed(self):
         title = library.name
         lane = CrawlableCollectionBasedLane()
         lane.initialize(library)
-        return self._crawlable_feed(
-            library=library, title=title, url=url, lane=lane
-        )
+        return self._crawlable_feed(title=title, url=url, lane=lane)
```

Comment:
`library` isn't in this method signature anymore. The test didn't catch it because the mock `_crawlable_feed` would take whatever arguments you gave it.